saas/models.py

```diff
@@ -1528,6 +1528,26 @@ def provider(invoiced_items):
             result = get_current_provider()
         return result
 
+    def by_organization(self, organization, transaction_type):
+        return self.filter(
+            # All transactions involving Funds
+            ((Q(orig_organization=organization)
+              & Q(orig_account=transaction_type))
+            | (Q(dest_organization=organization)
+              & Q(dest_account=transaction_type))))
+
+
+    def by_customer(self, organization):
```

Comment:
I was trying to follow the existing precedent set by `by_charge` and `by_subscriptions`: https://github.com/djaodjin/djaodjin-saas/blob/4845e7b4f32c86e2d0d4d07a1ac03b40cc1ee57b/saas/models.py#L1341-1357

I personally think that there are already a lot of financial terms in play here ("transactions," "funds," "withdrawals," "accounts," "cart item," and "plan item"), and I'm not really certain which is responsible for precisely what - for instance, I was surprised to find out that one must consult a "cart item" to find out how much a previous purchase cost. Which isn't to say that I object, but instead, just that I'm not really clear on everything.